lib/ansible/modules/monitoring/zabbix/zabbix_user_group.py

```diff
@@ -0,0 +1,331 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, OVH SAS
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: zabbix_user_group
+short_description: Create/update/delete Zabbix user group
+description:
+    - Create/update/delete Zabbix user group.
+version_added: "2.10"
+author:
+    - Emmanuel Riviere (@emriver)
+requirements:
+    - "python >= 2.7"
+    - "zabbix-api >= 0.5.3"
```

Comment:
```suggestion
    - "zabbix-api >= 0.5.4"
```
as `zbx.logout()` wasn't present in `0.5.3`.